docs/topics/mpp/multiplatform.md

```diff
@@ -7,68 +7,70 @@
 Support for multiplatform programming is one of Kotlin’s key benefits. It reduces time spent writing and maintaining the
  same code for [different platforms](mpp-supported-platforms.md) while retaining the flexibility and benefits of native programming. 
 
-This is how Kotlin Multiplatform works.
+## How Kotlin Multiplatform works
```

Comment:
Not sure if we need this title. For me it's a part of the introduction

Reply:
Agree

Reply:
I left the title and move this section below (instead of use cases)